src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -314,7 +324,7 @@ protected void setOtherFields(Variant variant, String fileId, String studyId, Se
         variant.getSourceEntry(fileId, studyId).addAttribute("src", line);
     }
 
-    protected void parseInfo(Variant variant, String fileId, String studyId, String info, int numAllele) {
+    protected void parseInfo(Variant variant, String fileId, String studyId, String info, int numAllele)throws NotAVariantException {
```

Comment:
Add a whitespace before `throws`, and break the line if it is longer than 120 characters.